frontend/src/routes.js

```diff
@@ -48,6 +49,10 @@ export const ROUTES = {
     account: {
       path: "/account",
       component: () => "Placeholder account"
+    },
+    createYourAccount: {
```

Comment:
I would use the same route as for account, not a new one. Can you please use the `/account` route?